src/utils/interfaces.ts

```diff
@@ -185,3 +185,44 @@ export interface DocumentHit extends Document {
    */
   _score: number;
 }
+
+export interface MappingsProperties {
+  /**
+   * Properties types definition
+   *
+   * @see https://docs.kuzzle.io/core/2/guides/essentials/database-mappings/#properties-types-definition
+   */
+  properties?: MappingsProperties,
+  /**
+   * Dynamic mapping policy
+   *
+   * @see https://docs.kuzzle.io/core/2/guides/essentials/database-mappings/#dynamic-mapping-policy
+   */
+  dynamic?: 'true' | 'false' | 'strict' | boolean
+}
+
+/**
+ * Collection mappings definition
+ *
+ * @see https://docs.kuzzle.io/core/2/guides/essentials/database-mappings/
+ */
+export interface CollectionMappings {
+  /**
+   * Collection metadata
+   *
+   * @see https://docs.kuzzle.io/core/2/guides/essentials/database-mappings/#collection-metadata
+   */
+  _meta?: JSONObject;
+  /**
+   * Properties types definition
+   *
+   * @see https://docs.kuzzle.io/core/2/guides/essentials/database-mappings/#properties-types-definition
+   */
+  properties?: MappingsProperties,
+  /**
+   * Dynamic mapping policy
+   *
+   * @see https://docs.kuzzle.io/core/2/guides/essentials/database-mappings/#dynamic-mapping-policy
+   */
+  dynamic?: 'true' | 'false' | 'strict',
```

Comment:
```suggestion
  dynamic?: 'true' | 'false' | 'strict' | boolean,
```

Reply:
Weird because I already add it here https://github.com/kuzzleio/sdk-javascript/pull/531/commits/725bf913e47b37d5fb7568b54323167db717c03a :thinking: